python-avd/schema_tools/generate_docs/yamllinegen.py

```diff
@@ -128,10 +127,7 @@ def render_field(self) -> Generator[YamlLine]:
             self.get_required(),
         ]
         # TODO: Remove legacy output
```

Comment:
Should this be a task of 5.0.0? we are not tracking it in https://github.com/aristanetworks/avd-internal/issues/157

Reply:
We have a task of evaluating all TODOs... All the handling of LEGACY_OUTPUT can be removed, since it was set to false above. It was not a way of having a simple change to modify all the docs in a single PR (requiring less review)